plugins/woocommerce/client/admin/client/blueprint/settings/slotfill.js

```diff
@@ -165,12 +152,15 @@ const Blueprint = () => {
 			<p className="export-intro">
 				{ createInterpolateElement(
 					__(
-						'Export your blueprint schema. Select the options you want to export, then click on "Export". Want to know more? <docLink/> ',
+						'Export your blueprint schema. Select the options you git want to export, then click on “Export”. Want to know more? <docLink/> ',
```

Comment:
Not intended I assume
```suggestion
						'Export your blueprint schema. Select the options you want to export, then click on “Export”. Want to know more? <docLink/> ',
```

Reply:
🤣